src/construction/constructors.rs

```diff
@@ -1,68 +1,50 @@
-use crate::data::{ApInt, Digit, DigitRepr};
-use crate::info::{BitWidth, Error, Result};
+use crate::{
+    data::{ApInt, Digit, DigitRepr},
+    info::{BitWidth, Error, Result},
+};
 
 use smallvec::SmallVec;
 
 /// # Constructors
 impl ApInt {
     /// Creates a new `ApInt` from the given boolean value with a bit width of `1`.
-    pub fn from_bool(bit: bool) -> ApInt {
-        ApInt::new_inl(BitWidth::w1(), Digit(bit as DigitRepr))
-    }
+    pub fn from_bool(bit: bool) -> ApInt { ApInt::new_inl(BitWidth::w1(), Digit(bit as DigitRepr)) }
```

Comment:
I won't comment the following auto-formatting issues but to be honest I really dislike these single super long lines. I am very much in favor of short lines that make it easy to follow along and review in PRs.